test-cases/manager/manager-backup-1TB-gce.yaml

```diff
@@ -0,0 +1,41 @@
+test_duration: 600
+
+user_prefix: 'mgr-backup-1tb'
+cluster_backend: 'gce'
+
+prepare_write_cmd: ["cassandra-stress write cl=QUORUM n=550100150 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native connectionsPerHost=64 -rate threads=150 throttle=45000/s -col 'size=FIXED(200) n=FIXED(5)' -pop seq=1..550100150",
+                    "cassandra-stress write cl=QUORUM n=550100150 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native connectionsPerHost=64 -rate threads=150 throttle=45000/s -col 'size=FIXED(200) n=FIXED(5)' -pop seq=550100151..1100200300",
+                    ]
+stress_read_cmd: ["cassandra-stress read cl=QUORUM duration=580m -port jmx=6868 -mode cql3 native  connectionsPerHost=64 -rate threads=150 -pop seq=1..1100200300  -log interval=5 -col 'size=FIXED(200) n=FIXED(5)'",
+                 ]
+round_robin: true
+
+pre_create_schema: true
+compaction_strategy: 'NullCompactionStrategy'
+post_prepare_cql_cmds: "ALTER TABLE keyspace1.standard1 WITH compaction = {'class': 'SizeTieredCompactionStrategy'}"
```

Comment:
One more thing I just thought of...
It's not enough to just change the compaction strategy here, but you also need to run "nodetool compaction" here.

Reply:
are you sure ? since I have a sample of it working as expected, without calling compaction.

Reply:
It switches the compaction strategy from now on, but not going back and compacting all previous sstables.

Reply:
o.k. I'll add to the wait for compaction, to call `nodetool compat` before waiting.

Reply:
fixed